### DISCUSSION TITLE
Can we send without HTML

### DISCUSSION THREAD

**Discussion by ghost** [PHPMailer/PHPMailer#2945] [Q&A] [answered]
It works if I didn't specify the `AltBody`:
`$mail->isHTML(true); $mail->Body  = 'This is the HTML message body <b>in bold!</b>';`

But what if I want to send without HTML ( I'm sure the content should only be plain text ):
`$mail->AltBody = 'This is the body in plain text for non-HTML mail clients';`

This throws an error saying that the `Body` is missing.

Another question, do `$mail->AltBody` required / necessary when I only want to send HTML content like the first example?

**Comment by Synchro** [comment, marked as answer]
If you want to set plain-text only, that's the default behaviour, so it's very simple:

    $mail->Body = 'my message body';

`Body` is a plain text body by default, but if you call `isHTML`, it will be handled as HTML, so this will produce an HTML-only message:

    $mail->isHTML();
    $mail->Body = '<b>my HTML message body</b>';

`AltBody` is there to provide a fallback to plain text, and typically you only set that if you are also using HTML:

    $mail->isHTML();
    $mail->Body = '<b>my HTML message body</b>';
    $mail->AltBody = 'my plain text message body';

The `msgHtml()` method provides a shortcut and some automation to this process:

    $mail->msgHtml('<b>my HTML message body</b>');

This will call `isHTML`, set `Body` to the HTML you provide (after some processing to handle images), and strip tags and use it to set `AltBody`.

**Comment by ghost** [comment]
@Synchro May I know how to add line break for plain-text for ( `$mail->Body` without `$mail->isHTML()` ) and ( `$mail->AltBody` ).

**Reply by Synchro** [reply]
Lin breaks will be ignored in rendered HTML - you need to use `p` and `br` tags to add them. In plain text, line breaks will be preserved as in the original text, so you should not have to do anything.
